__tests__/MoreScreen.test.tsx

```diff
@@ -106,14 +105,70 @@ describe('MorePage', () => {
       "mailto:yashkamal.bhatia@slu.edu?subject=Bug%20Report%20on%20'Where's%20Religion%3F'&body=Please%20provide%20details%20of%20your%20issue%20you%20are%20facing%20here."
     );
   });
-  it('renders the "Logout" button', () => {
-    const { getByText } = render(
+
```

Comment:
Good test cases and good comments. However some comments can be better, like comment on what it is testing and what the value should be.